benchmarks/benchmark_hf.sh

```diff
@@ -22,7 +22,7 @@ if [[ $framework == transformers ]]; then
     fi
     ver=`pip show transformers | awk  '{if($1=="Version:")print $2}'`
     framework_versioned="transformers_v$ver"
-    util="python $BASELINE_REPO/examples/seq2seq/run_eval.py"
+    util="python run_eval_hf.py"
```

Comment:
why change here? Looks we don't need to copy a file and add to FastSeq git repo, Transformers is cloned and saved to local, so we can use the file there.

Reply:
`run_eval.py` in huggingface-transformers repo does not support gpt2 model. This file adds some changes: 1) line 73-79: use `AutoModelForCausalLM` to load gpt2 model and specify the pad token; 2) line 93, 103, 135-140, 156-157 are added to support max_tokenizer_length and max_gen_length parameters.